### DISCUSSION TITLE
Always get "[WARN ] Please restart your console after complete."

### DISCUSSION THREAD

**Discussion by jimmckeeth** [HashLoad/boss#172] [Q&A]
Just using Boss again after a while. I keep getting the message `[WARN ] Please restart your console after complete.` even when I've just restarted the console. Am I missing something or did I install wrong?

Instructions say to download **setup** and run `boss`, which didn't seem to install the boss.exe, so I put it on the path. Still getting the **`[WARN ]`** message though.

**Comment by snakeice** [comment]
This happens because of this MR #137 , basically it sets the env, but not persistently after the merge was done, I will analyze to revert this modification
